xikolo-common/Data/Routes.swift

```diff
@@ -16,6 +16,8 @@ class Routes: NSObject {
     
     static let HEADER_USER_PLATFORM = "User-Platform"
     static let HEADER_USER_PLATFORM_VALUE = "iOS"
+    static let HEADER_AUTHORIZATION = "AUTHORIZATION"
+    static let HEADER_AUTHORIZATION_VALUE_SCHEMA = "Token token="
```

Comment:
Why do you need this again? See `HTTP_AUTH_HEADER` and `HTTP_AUTH_HEADER_VALUE_PREFIX` below.